css/src/utils/utility.css

```diff
@@ -34,3 +34,230 @@
 .bottom-0 {
   bottom: 0;
 }
+
+/** New utility classes **/
+
+/* Visibility */
+.visible {
+  visibility: visible !important;
+}
+
+.invisible {
+  visibility: hidden !important;
+}
```

Comment:
Please change this classname to something else like: `visibility-0` & `visibility-1`